### ISSUE TITLE
quick question

### ISSUE THREAD

**Issue by Mayonese** [Monika-After-Story/MonikaModDev#4946]
Monika's black dress only appears in special days or when ur affection is 1000+ she has in the wardrobe?

**Comment by multimokia** [comment]
You needed to take her out on her birthday

**Comment by multimokia** [comment]
No. You could only unlock it via her birthday. If she wore it when you went out, it'll be unlocked already at 1000+ affection. Otherwise no.
